Makefile

```diff
@@ -75,3 +75,6 @@ docker_push: # Push new docker image to docker hub
 
 docker_login: # Login to docker hub
 	docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD}
+
+cs_fixer: # Runs php-cs-fixer
+	docker-compose -f ${dc_path} exec ${app_container} vendor/bin/php-cs-fixer fix --config=./.php_cs
```

Comment:
Hello @korzol 

Does it work with ```./vendor/bin/....```?

If yes then could make the command with leading leading dot in order to use the same stile we are using in the make file for other commands?

Reply:
Hello @ceckoslab 
Yes, it does the work even without ./  But anyway I changed it accordingly
```
make cs_fixer
docker-compose -f docker/docker-compose.yml exec symfony_app ./vendor/bin/php-cs-fixer fix --config=./.php_cs
Loaded config default from "./.php_cs".
Using cache file ".php_cs.cache".
   1) src/Controller/BeaconController.php

Fixed all files in 0.029 seconds, 12.000 MB memory used
```